lib/rollbar/request_data_extractor.rb

```diff
@@ -57,6 +57,26 @@ def extract_request_data_from_rack(env)
       data
     end
 
+    def extract_custom_data_from_rack(env)
+      controller = env["action_controller.instance"]
+      if !controller
+        Rollbar.configuration.custom_data_method.call
```

Comment:
So here we have a `Rack` app and not a `Rails` app, probably we'll want to pass `env` to `#call`?